_spinnaker/using_dinghy.md

```diff
@@ -432,3 +432,70 @@ If you have already created a pipeline in the Spinnaker UI, you can create a din
 Save this file as `dinghyfile` in the root of your project and push it to your repository.
 
 You may want to follow the [deleting stale pipelines](http://localhost:4000/spinnaker/using_dinghy/#deleting-stale-pipelines).
+
+# Alternate Template Formats
+
+When using an alternate template format all of your modules must also be in that same format.
+
+## YAML Template Format
+
+YAML formatting works just like the JSON formatting does.  However, all of your templates will need to be YAML if you've configured dinghy to use YAML as its template fomat.
+
+Example:
+```yaml
+---
+application: "My awesome application"
+# You can do inline comments now
+globals:
+  waitTime: 42
+  retries: 5
+pipelines:
+- application: "My awesome application"
+  name: "My cool pipeline"
+  appConfig: {}
+  keepWaitingPipelines: false
+  limitConcurrent: true
+  stages:
+  - name: Wait For It...
+    refId: '1'
+    requisiteStageRefIds: []
+    type: wait
+    waitTime: 4
+  {{ module "some.stage.module" "something" }}
+  triggers: []
+```
+
+*Note: YAML has strict spacing requirements.  Your modules must indent properly for the template to be rendered correctly.*
+
+## HCL Template Format
+
+```
+"application" = "Some App"
+"globals" = {
+    "waitTime" = 42
+}
+"pipelines" = [
+  {
+    "appConfig" = {}
+    "application" = "Some App"
+    "keepWaitingPipelines" = false
+    "limitConcurrent" = true
+    "name" = "Foo"
+    "stages" = [
+        {
+          "name" = "Wait For It..!"
+          "refId" = "1"
+          "requisiteStageRefIds" = []
+          "type" = "wait"
+          "waitTime" = 5
+        },
+        { 
+            {{ module "some.stage.module" "something" }} 
```

Comment:
This line, at least needs to be escaped, `{% raw %}` and `{% endraw %}` should do it.